src/Language/PureScript/Docs/Convert.hs

```diff
@@ -3,28 +3,27 @@
 
 module Language.PureScript.Docs.Convert
   ( convertModules
+  , convertModulesWithEnv
   , convertModulesInPackage
-  , collectBookmarks
+  , convertModulesInPackageWithEnv
   ) where
 
-import Prelude.Compat
+import Prelude (id, String)
```

Comment:
Just in case you didn't know `Protolude` exports `id` as `identity` to free up the name `id`.